api/v1alpha1/alluxiocluster_types.go

```diff
@@ -257,14 +257,24 @@ type AlluxioMonitorSpec struct {
 type EtcdSpec struct {
 	Enabled      *bool             `json:"enabled" yaml:"enabled"`
 	Auth         EtcdAuthSpec      `json:"auth,omitempty" yaml:"auth,omitempty"`
-	Image        EtcdImageSpec     `json:"image,omitempty" yaml:"image,omitempty"`
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" yaml:"nodeSelector,omitempty"`
 	ReplicaCount int               `json:"replicaCount,omitempty" yaml:"replicaCount,omitempty"`
+	Resources    ResourcesSpec     `json:"resources,omitempty" yaml:"resources,omitempty"`
```

Comment:
what does this field do?

Reply:
Specify the amount of memory and cpu that the etcd container uses